packages/discussions/src/types.ts

```diff
@@ -441,7 +441,10 @@ export enum PostType {
  * @extends {IWithEditor}
  * @extends {IWithTimestamps}
  */
-export interface IPost extends IWithAuthor, IWithEditor, IWithTimestamps {
+export interface IPost
+  extends Partial<IWithAuthor>,
+    Partial<IWithEditor>,
```

Comment:
Wrapping `IWithAuthor` and `IWithEditor` is going to change the `IPost` interface such that `creator` and `editor` fields become optional. This is a breaking change, so we'll need to make sure that is reflected in the commit message(s) so things get published correctly when this merges. I believe the `npm run c` command walks you through this, see  [CONTRIBUTING.md](https://github.com/Esri/hub.js/blob/f/post-supports-anon/docs/CONTRIBUTING.md#formatting-commit-messsages).

Reply:
Thanks @rweber-esri for pointing this out.  I pushed the changes up with the breaking changes comment using `npm run c`.